build.sh

```diff
@@ -28,12 +28,19 @@ fi
 
 if [ $(DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $build_image_name 2> /dev/null ; echo $?) -eq 0 ]; then
 	echo "----> Skipping build as image already exists"
-else
+	exit 0
+fi
+
+if [ -z "$WORKLOAD_PATH" ]; then
+        $WORKLOAD_PATH=.
```

Comment:
@fagiani here `$WORKLOAD_PATH` receive the "." as value if it's empty, thinking about this with @fagianijunior we think wich is unnecessary utilize bash parameter expansion to change the value to "." again.